Makefile

```diff
@@ -1,7 +1,7 @@
 CC = gcc
-CFLAGS = -g -Wall -Werror -fPIC -std=gnu99
+CFLAGS = -g -Wall -Werror -fPIC -std=gnu99 -DTESTING
```

Comment:
Do you actually want -DTESTING unconditionally enabled in 'make' ?  or was this meant to stay in a testing branch?